apis/node/version.yaml

```diff
@@ -10,6 +10,9 @@ get:
       content:
         application/json:
           schema:
-            $ref: '../../beacon-node-oapi.yaml#/components/schemas/Version'
+            type: object
+            properties:
+              data:
+                $ref: '../../beacon-node-oapi.yaml#/components/schemas/Version'
```

Comment:
I read the doc as this should return
``` 
{
  "data": {
   "version": String
  }
}
```
I could be mis-understanding, thoughts? currently, this returns
```
{
  "data": "Lighthouse / v0.1.5 (Linux x86_64)"
}
```
and the spreadsheet indicates:
```
data {
  version string
}
```
in the response...